src/SleepRepository.js

```diff
@@ -1,36 +1,16 @@
 // import Sleep from "../src/Sleep";
 
 class SleepRepository {
-  constructor(todaysDate) {
+  constructor() {
     this.individualEntryRecords = [];
-    // this.hoursSlept = this.findTodaysTotalSleep(todaysDate);
-    // this.weeklyHoursSlept = this.getWeekAveHoursSlept()
-    // this.weeklyQualHoursSlept = this.calculateAverageHoursThisWeek(todaysDate)
   }
   
-  findLastNightsSleepQual(date) {
-    console.log(this.individualEntryRecords[0])
+  findLastNightsSleepQual() {
     return this.individualEntryRecords[0].sleepQuality
-    // let QualityHrsSleptToday = this.individualEntryRecords.filter(record => {
-    //   return record.date === date;
-    // })
-    //   .reduce((sum, entry) => {
-    //     sum += entry.hoursSlept;
-    //     return sum;
-    //   }, 0);
-    //   console.log(QualityHrsSleptToday)
-    // return QualityHrsSleptToday;
   }
 
-  findLastNightsHoursSlept(date) {
-    let hoursSleptToday = this.individualEntryRecords.filter(record => {
-      return record.date === date;
-    })
-      .reduce((sum, entry) => {
-        sum += entry.hoursSlept;
-        return sum;
-      }, 0);
-    return hoursSleptToday;
+  findLastNightsHours() {
+    return this.individualEntryRecords[0].hoursSlept
   }
 
   getWeeksDailyHours() {
```

Comment:
What is this line for? I can't see where the closing curly braces are

Reply:
This is technically supposed to be getting Todays stats, but just how sleeping works, you wake up and input your data, so that stat, so on the card, it'll say something like, "Last Night You Slept for X Hours". Is that clear? I can explain it on tuple as well